src/screens/FeedScreen/index.js

```diff
@@ -236,7 +239,8 @@ const FeedScreen = (props) => {
     sendViewPostTime(true);
 
     navigation.navigate('DiscoveryScreen', {
-      tab: DISCOVERY_TAB_TOPICS
+      tab: DISCOVERY_TAB_TOPICS,
+      initialTopics: topics
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Passing `initialTopics` as a navigation parameter seems fine. However, ensure that the `topics` data isn't too large to avoid potential performance issues related to passing large amounts of data between screens.
```diff
- initialTopics: topics
+ initialTopics: topics.slice(0, MAX_TOPICS_TO_PASS)
```
In this case, `MAX_TOPICS_TO_PASS` should be a constant that limits the number of topics passed to avoid potential performance issues.


<!-- This is an auto-generated comment by OSS CodeRabbit -->